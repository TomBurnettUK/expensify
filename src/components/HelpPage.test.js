import React from 'react';
import { shallow } from 'enzyme';
import HelpPage from './HelpPage';

it('should render correctly', () => {
  const wrapper = shallow(<HelpPage />);
  expect(wrapper).toMatchSnapshot();
});
