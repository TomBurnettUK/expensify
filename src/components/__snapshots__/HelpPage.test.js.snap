// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <p>
    This is from the help component
  </p>
</div>
`;
